platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -2671,6 +2671,7 @@ void DisplayServerX11::cursor_set_custom_image(const Ref<Resource> &p_cursor, Cu
 		Ref<Image> image = texture->get_image();
 
 		ERR_FAIL_COND(!image.is_valid());
+		ERR_FAIL_COND_MSG(image->is_compressed(), "Only non-VRAM-compressed formats are supported for custom mouse cursors. To resolve this, change the cursor texture's compress mode to Lossless in the Import dock then click Reimport.");
```

Comment:
Image should be usable if the specific format is decompressible (everything except ETC), so probably it can be something like:

```c++
if (image->is_compressed()) {
    image = image->duplicate(true);
    Error err = image->decompress();
    ERR_FAIL_COND_MSG(err != OK, "........");
}
```